COD60276.TXT

```diff
@@ -0,0 +1,237 @@
+OBJECT Codeunit 60276 Api Mgt.
+{
+  OBJECT-PROPERTIES
+  {
+    Date=01-09-17;
+    Time=12:34:41;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+
+    LOCAL PROCEDURE "--- Init Request"@6014407();
+    BEGIN
+    END;
+
+    PROCEDURE InitWebRequest@9(Uri@6014403 : DotNet "'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Uri";Username@6014400 : Text;Password@6014401 : Text;Domain@6014402 : Text;VAR HttpWebRequest@1000 : DotNet "'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Net.HttpWebRequest");
```

Comment:
Either make the procedure local or remove the DotNet types from the parameter list, as the function cannot be called from the extension.